### PR TITLE
User refactor

### PR DESCRIPTION
**Pull request by thebadpanda**

### CODE REVIEW

**Review by md1guy — DISMISSED** [review]

**Review by octap16 — DISMISSED** [review]

File: src/main/java/com/softserve/actent/controller/EquipmentController.java [hunk 1]
```diff
@@ -64,17 +64,17 @@ public EquipmentController(EquipmentServiceImpl equipmentServiceImpl, UserReposi
         Equipment newEquipment = modelMapper.map(createEquipmentDto, Equipment.class);
 
         // TODO: use User and Event services for check if they exists
-        User user = userRepository.getOne(createEquipmentDto.getUserid());
-        Event event = eventRepository.getOne(createEquipmentDto.getEventid());
+        User user = userRepository.getOne(createEquipmentDto.getUserId());
+        Event event = eventRepository.getOne(createEquipmentDto.getUserId());
```
Comment (Pitcity) [inline src/main/java/com/softserve/actent/controller/EquipmentController.java]: you're fetching event by userId? Could the user have more than 1 event assigned?

Comment (thebadpanda) [reply src/main/java/com/softserve/actent/controller/EquipmentController.java]: damn. thanks)

**Review by md1guy — DISMISSED** [review]

**Review by octap16 — DISMISSED** [review]

**Review by Senchakmen — DISMISSED** [review]

**Review by Senchakmen — DISMISSED** [review]

**Review by onlyVone-zz — DISMISSED** [review]

**Review by md1guy — DISMISSED** [review]

**Review by RBarnabass — DISMISSED** [review]

**Review by RBarnabass — DISMISSED** [review]

**Review by Senchakmen — DISMISSED** [review]

File: src/main/java/com/softserve/actent/controller/UserController.java [hunk 2]
```diff
@@ -29,34 +43,59 @@ public UserController(ModelMapper modelMapper, UserService userService) {
     }
 
     @PostMapping(value = "/users")
-    public ResponseEntity<UserDto> registerUser(@RequestBody RegisterUserDto registerUserDto) {
+    @ResponseStatus(HttpStatus.CREATED)
+    public IdDto addUser(@Validated @RequestBody RegisterUserDto registerUserDto, BindingResult bindingResult) {
+
+        // TODO: change to better practise
+        if (bindingResult.hasErrors()){
```
Comment (Senchakmen) [inline src/main/java/com/softserve/actent/controller/UserController.java]: Its useless we use handleMethodArgumentNotValid in RestExceptionHandler

**Review by RBarnabass — APPROVED** [review]

File: src/main/java/com/softserve/actent/model/entity/User.java [hunk 3]
```diff
@@ -19,36 +30,35 @@
...
     @NotBlank(message = StringConstants.EMPTY_USER_EMAIL)
-    @Column(unique = true, nullable = false)
+    @Column(unique = true, nullable = false, length = 40)
     @Email
     private String email;
 
     @NonNull
     @NotBlank(message = StringConstants.EMPTY_USER_PASSWORD)
-    @Column(nullable = false)
+    @Column(nullable = false, length = 30)
```
Comment (Senchakmen) [inline src/main/java/com/softserve/actent/model/entity/User.java:61]: Remove magic variables

**Review by nazarmavko — APPROVED** [review]

**Review by octap16 — APPROVED** [review]

**Review by Senchakmen — APPROVED** [review]

**Pull request merged**